map_server/CMakeLists.txt

```diff
@@ -60,10 +100,20 @@ if(CATKIN_ENABLE_TESTING)
   catkin_add_gtest(${PROJECT_NAME}_utest test/utest.cpp test/test_constants.cpp)
   target_link_libraries(${PROJECT_NAME}_utest image_loader SDL SDL_image)
 
-  add_executable(rtest test/rtest.cpp test/test_constants.cpp)
+  add_executable(rtest
```

Comment:
while we are at it -- we should make this name more specific by prefacing with map_server_